package.json

```diff
@@ -19,7 +19,7 @@
     "test:watch": "watchy -w src/ -w custom_typings/ -- npm run test:fast --loglevel=silent",
     "test": "npm run test:fast && npm run lint",
     "lint": "tslint -p ./",
-    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
+    "format": "find bin src test | grep \"\\.js$\\|\\.ts$\" | xargs clang-format --style=file -i",
```

Comment:
This command will properly run on windows now too.

Reply:
nice!